File: trunk/bms/src/main/java/com/bms/resource/enumeration/DictType.java
package com.bms.resource.enumeration;


public enum DictType {
	
	CATEGORY("分类"),
	
	PRESS("出版社");
	
	private String desc;
	
	DictType(String desc) {
		this.desc = desc;
	}
	
	public String getDesc() {
		return desc;
	}
}

File: trunk/bms/src/main/webapp/static/js/resource/DictType.js
Ext.define('Bms.resource.DictType', {
	extend : 'Ext.container.Container',
	
	initComponent : function() {
		var me = this;
		
		var store = Ext.create('Ext.data.Store', {
			autoLoad : true,
		    fields:['type', 'desc'],
		    proxy: {
		    	type: 'ajax',
		        url: Base.calUrl('resource/dictType/list')
		    }
		});
		
		Ext.apply(me, {
			layout : 'fit',
			items : [{
				xtype : 'grid',
				store : store,
				columns: [
				   { text: '字典代码',  dataIndex: 'type', flex: 1 },
		           { text: '描述', dataIndex: 'desc', flex: 1 }
		        ]
			}]
		});
		
		me.callParent();
	}
});